models/staging/edfi_3/stage/stg_ef3__grades__learning_standards.sql

```diff
@@ -0,0 +1,15 @@
+with stg_grades as (
+    select * from {{ ref('stg_ef3__grades') }}
+    where not is_deleted
+),
+flattened as (
+    select
+        stg_grades.* exclude(letter_grade_earned, numeric_grade_earned),
```

Comment:
I wonder if we should keep these, in cases where both might be filled in, so those overall grades can be in the fact table.

For example, student took 4th grade reading, get's a B overall, but also has a 3 & 4 in the two learning standards. That may not be true in Boston, but is it in JeffCo? Could it be possible elsewhere?

I'm torn whether it's best to keep that info only in fct_grades, or if it should be in the fct_learning_standard_grades, too.

Reply:
I am checking this with Susan atm. I don't see a way of verifying this through the Ed-Fi documentation, in a way even if JeffCo doesn't have cases where there is a grade and a learning standard grade for a single record, we could still include those columns since we can't verify this never being the case in other installations. It's probably safer to not make assumptions based on only two projects?

Reply:
Susan shared that there are no cases in Jeffco where a record contains both a grade and a learning standard grade.

Reply:
ok I talked with Jules about this and I take back my earlier comment! I think excluding those grades is the correct thing to do, because otherwise, there will be duplicated information that could lead to incorrect e.g. gpa calcuation.

She also suggested:
1. keep even fewer columns in this stg table -- only the columns that will be pulled into the fact table
2. potentially create a k_ for k_student_grade, as the natural key of fct_student_grades, and available in fct_student_learning_standard_grades to make it easy to join the two tables together if needed

We don't yet have a use case for 2., but we might some day

Reply:
Thank you both for the comments, I just finished addressing them. 
I also renamed `letter_grade_earned` to` learning_standard_letter_grade_earned` for future clarity, since I thought having 2 columns with the same exact name but representing two different things might be confusing. Let me know if the name is too long or if it is not a necessary change and I can undo this fairly quickly!